etc/workloads/service-architecture/PrimaryRequiredReads.yml

```diff
@@ -0,0 +1,67 @@
+SchemaVersion: 2018-07-01
+Pool:
+    QueryOptions:
+        maxPoolSize: 1000
+        socketTimeoutMS: 5000
+        connectTimeoutMS: 5000
+Actors:
+  - Name: StepdownCommand
+    Type: AdminCommand
+    Threads: 1
+    # So this *will* fail the first time because that's what replSetStepdown does.
```

Comment:
1. what do you mean "fail the first time"?
2. I think add a comment that this `ExecutionStrategy` block basically means that the Actor won't throw on exceptions encountered when running the command.

Reply:
Yeah, I can document exactly what the ExecutionStrategy means here. I mean, I have a key below that says "ThrowOnFailure". I'd be pretty disappointed if people got stuck on my wording.